dev-requirements.txt

```diff
@@ -4,5 +4,4 @@ pytest-qt
 pytest-cov
 pytest-timeout
 p4p
-pyca
```

Comment:
Is there a reason for moving pyca from `dev-requirements.txt` to `requirements.txt`? I think it makes sense to remain an optional dependency - the user would only need to install it if they want to use it for making EPICS connections instead of pyepics.

Reply:
Description of `pyproject.toml` in [`setuptools-scm`](https://setuptools-scm.readthedocs.io/en/stable/usage/).

Example of use in the `gemviz` application here: https://github.com/BCDA-APS/gemviz/blob/17ca7f896222b40df0e1fac3b68b570fb09aa002/pyproject.toml#L48-L76